src/ui/units/wizard/containers/Wizard/Wizard.tsx

```diff
@@ -171,7 +171,7 @@ class Wizard extends React.Component<Props, State> {
         this.props.initEditHistoryUnit({
             unitId: WIZARD_EDIT_HISTORY_UNIT_ID,
             setState: ({state}) => {
-                return this.props.setWizardStore({
```

Comment:
Now setState handler here does not return action, because all actions are dispatched inside